### PR TITLE
Add nnord option for measuring noise cov

### PR DESCRIPTION
**Pull request by andyyPark**

### CODE REVIEW

File: bin/fpfs_process_sim.py [hunk 1]
```diff
@@ -13,17 +13,18 @@
...
 import json
-import schwimmbad
-import numpy as np
-import astropy.io.fits as pyfits
+import os
 from argparse import ArgumentParser
 from configparser import ConfigParser
 
+import astropy.io.fits as pyfits
+import fpfs
```
Comment (mr-superonion) [inline bin/fpfs_process_sim.py]: Any reason putting so many empty lines?
Why not use the old version which takes less line and looks better?

**Review by mr-superonion — CHANGES_REQUESTED** [review]

**Comment by mr-superonion** [comment]
Also, please merge from the main branch to solve the conflicts and then I will merge this PR.

File: bin/fpfs_process_sim.py (hunk 1 above)
Comment (andyyPark) [reply bin/fpfs_process_sim.py]: Hmm this change was by black I think? Or it could be from my vim...

Comment (andyyPark) [reply bin/fpfs_process_sim.py]: Fixed so that it matches with the one from main branch

**Review by mr-superonion — APPROVED** [review]

**Pull request merged**